rpc/api/eth.go

```diff
@@ -166,6 +167,20 @@ func (self *ethApi) IsMining(req *shared.Request) (interface{}, error) {
 	return self.xeth.IsMining(), nil
 }
 
+func (self *ethApi) IsSyncing(req *shared.Request) (interface{}, error) {
```

Comment:
Yeah, but I tried to follow the `IsMining` convention and the `syncing` property name "standardized" by @frozeman. I agree it's not really the best name, but it's probably now or never if we release it as part of the RPC spec.